modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -527,7 +527,15 @@ func (suite *KeeperTestSuite) TestChanUpgradeAck() {
 		{
 			"fails due to upgrade incompatibility",
 			func() {
+				// Need to set counterparty upgrade in state and update clients to ensure
```

Comment:
I've done a similar thing for the timeout in https://github.com/cosmos/ibc-go/pull/4305, lmk if there's a better way to go about it